paddle/fluid/operators/spectral_norm_op.cc

```diff
@@ -1,173 +0,0 @@
-/* Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserve.
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-   http://www.apache.org/licenses/LICENSE-2.0
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License. */
-
-#include <memory>
-
-#include "paddle/fluid/framework/infershape_utils.h"
-#include "paddle/fluid/framework/op_registry.h"
-
-#include "paddle/phi/infermeta/backward.h"
-#include "paddle/phi/infermeta/ternary.h"
-
-namespace paddle {
-namespace operators {
-
-class SpectralNormOp : public framework::OperatorWithKernel {
- public:
-  using framework::OperatorWithKernel::OperatorWithKernel;
-
- protected:
-  phi::KernelKey GetExpectedKernelType(
-      const framework::ExecutionContext& ctx) const override {
-    return phi::KernelKey(
-        OperatorWithKernel::IndicateVarDataType(ctx, "Weight"), ctx.GetPlace());
-  }
-};
-
-class SpectralNormOpMaker : public framework::OpProtoAndCheckerMaker {
- public:
-  void Make() override {
-    AddInput("Weight",
```

Comment:
根据`R-CI-Static-Check`的输出，由于`def spectral_norm`的comment引用了该文件中的comment，所以需要将这里的comment拷贝到`def spectral_norm`的`${}`位置，对应的文件是`./python/paddle/static/nn/common.py`。
原始说明：
```
    Args:
        weight(Tensor): ${weight_comment}
        dim(int): ${dim_comment}
        power_iters(int): ${power_iters_comment}
        eps(float): ${eps_comment}
        name(str, optional): For detailed information, please refer
                             to :ref:`api_guide_Name`. Usually name is no need to set and
                             None by default.
```
修改为：
```
    Args:
        weight(Tensor): The input weight tensor of spectral_norm operator,
                        This can be a 2-D, 3-D, 4-D, 5-D tensor which is the
                        weights of fc, conv1d, conv2d, conv3d layer.
                        The data type is float32 or float64.
        dim(int): The index of dimension which should be permuted
                  to the first before reshaping Input(Weight) to
                  matrix, it should be set as 0 if Input(Weight) is
                  the weight of fc layer, and should be set as 1 if
                  Input(Weight) is the weight of conv layer, default 0.
        power_iters(int): number of power iterations to calculate spectral norm, default 1.
        eps(float): epsilon for numerical stability in calculating norms, it will be added to
                    the denominator to aviod divide zero. Default 1e-12.
        name(str, optional): For detailed information, please refer
                             to :ref:`api_guide_Name`. Usually name is no need to set and
                             None by default.
```
并且删除`def spectral_norm(weight, dim=0, power_iters=1, eps=1e-12, name=None):`的装饰器`@templatedoc()`
并且根据找相关TMP review
```
@sunzhongkai588 或 @Ligoml
```